src/api/users/users.router.js

```diff
@@ -3,6 +3,8 @@ import * as usersController from './users.controller.js';
 
 const router = Router();
 
-router.get('/:id', usersController.getById);
+router.get('/id/:id', usersController.getById);
+// updated getById path as the path "/profile" was being interpreted as a params of getById!!
```

Comment:
Comments shouldn't be uploaded to the main branch 😃

Reply:
Totalmente de acuerdo. Es un comentario útil, pero sólo útil en este momento, para explicar el por qué de un cambio. Cuando alguien vea este comentario de aquí 6 meses no va a entender por qué se puso.

Esto quiere decir que no es un comentario que deba ir en el código, porque no explica qué hace el código, si no por qué se hizo.

Este comentario tendría que ser parte de este PR, no del código.

Reply:
Resuelto!